src/Lecture1.hs

```diff
@@ -77,7 +80,16 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+{- todo why are Nums not Ords?
+   todo should it be Num b, Num c, Num d instead? too many ords and nums then
+-}
+minmax :: (Num a, Ord a) => a -> a -> a -> a
```

Comment:
You need `Num` to be able to subtract values. And you need `Ord` to compare them. These are two separate typeclasses.

I'm going to explain typeclasses in Lecture 3 🎞️